extensions/arc/src/ui/dashboards/postgres/postgresParametersPage.ts

```diff
@@ -166,20 +155,19 @@ export class PostgresParametersPage extends DashboardPage {
 							cancellable: false
 						},
 						async (_progress, _token): Promise<void> => {
-							//Edit multiple
-							// azdata arc postgres server edit -n <server group name> -e '<parameter name>=<parameter value>, <parameter name>=<parameter value>,...'
 							try {
-								this.engineSettingUpdates!.forEach((value: string) => {
-									this.engineSettings += value + ', ';
+								this.parameterUpdates!.forEach((value, key) => {
+									engineSettings.push(`${key}=${value}`);
```

Comment:
It would be good to verify the values are escaped correctly.  Can we set settings whose values contain spaces, equals, commas?

Reply:
Good call, will use quotation marks around the values for this.